picard/ui/coverartbox.py

```diff
@@ -238,8 +243,8 @@ def open_release_page(self):
         lookup = self.tagger.get_file_lookup()
         lookup.albumLookup(self.release)
 
-    def fetch_remote_image(self, url):
-        return self.parent().fetch_remote_image(url)
+    def fetch_remote_image(self, url, dropped_data):
+        return self.parent().fetch_remote_image(url, dropped_data)
 
```

Comment:
Reviewing this I noticed a problem in master, so I created https://github.com/metabrainz/picard/pull/665 . Can you review that PR and make the corresponding rebase here? (note that the PR removes fetch_remote_image from CoverArtThumbnail)